.travis.yml

```diff
@@ -7,12 +7,6 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_NPY=110  CONDA_PY=34
-    - CONDA_NPY=111  CONDA_PY=34
-    - CONDA_NPY=110  CONDA_PY=35
-    - CONDA_NPY=111  CONDA_PY=35
```

Comment:
That is what I have actually. I'll try rerendering on a different machine for now.